core/ui/BinaryWarning.tid

```diff
@@ -1,8 +1,4 @@
 title: $:/core/ui/BinaryWarning
 
 \define lingo-base() $:/language/BinaryWarning/
-<div class="tc-binary-warning">
```

Comment:
Why is this class being removed?

Reply:
This class is moved from [$:/core/ui/BinaryWarning.tid](https://github.com/Jermolene/TiddlyWiki5/pull/4423/files/43427900ddb11025c2db2a1ceb952e9252d490b8#diff-c58fb2aaa141e94ac70468fbad065ff8) to [$:/core/modules/parsers/binaryparser.js](https://github.com/Jermolene/TiddlyWiki5/pull/4423/files#diff-fc3d3179097ec63eeb5dd9a85c568920) and [$:/core/modules/widgets/edit-binary.js](https://github.com/Jermolene/TiddlyWiki5/pull/4423/files#diff-62ccbeda3fbb154516f949d819aa9178), the binary tiddler view and edit mode respectively, so that [$:/core/ui/BinaryWarning.tid](https://github.com/Jermolene/TiddlyWiki5/pull/4423/files/43427900ddb11025c2db2a1ceb952e9252d490b8#diff-c58fb2aaa141e94ac70468fbad065ff8) (now just the warning message text without the class) can be transcluded into a single instance of the class together with the download link in the binary tiddler view and edit modes. 

It was done in this way because I am not familiar with how the lingo macro and translation in general works, so I avoided messing with it. I have also previously tried a new 2nd instance of the class for the download link but it looked strange to me compared to this option of combining the message and download link in a single instance of the class. I would be interested to know any more elegant or idiomatic ways to achieve this as I am still quite new to TiddlyWiki.

Reply:
Hi @s-l-lee that makes perfect sense, and is a good solution, thank you.